docs/using-the-jdbc-driver/ReaderSelectionStrategies.md

```diff
@@ -0,0 +1,16 @@
+## Reader Selection Strategies
+By default, the read/write plugin randomly selects a reader instance the first time that `setReadOnly(true)` is called. To balance connections to reader instances more evenly, different selection strategies can be used. The following table describes the currently available selection strategies and any relevant configuration parameters for each strategy.
```

Comment:
Can we remove references about RW splitting plugin? I think reader (or in general, node) selection strategies are used driver wide, by many plugins.

Reply:
Removed the reference to the RW plugin